supervisor/dbus/udisks2/__init__.py

```diff
@@ -170,4 +230,5 @@ def shutdown(self) -> None:
             block_device.shutdown()
         for drive in self.drives:
             drive.shutdown()
+        self.udisks2_object_manager.shutdown()
```

Comment:
How fast is disconnecting the previous objects from the bus? Can we assume `on_interfaces_added` won't be called before the manager is shut down?

Reply:
From debug logs when attaching a disk, it looks like all is called in the same Thread and hence event loop, so it *probably* doesn't matter in what order these operations are executed. But logically it makes more sense to shutdown the object manager first, which stops the signal handling.

I've changed the order and tested shutdown, still works without errors.